py/desimodel/install.py

```diff
@@ -123,7 +133,7 @@ def main():
     options = parser.parse_args()
     try:
         status = install(options.desimodel, options.desimodel_version)
```

Comment:
Here, you eliminated the return value, but it is still being assigned to a variable.